### PR TITLE
Solved CSS-grid place elements

### PR DESCRIPTION
**Pull request by WinterIsCoding23**

### CODE REVIEW

File: css-grid_place-elements/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+}
+
+.element-2 {
+  background-color: #3a67a4;
+  /* Add styles here */
+  grid-column-start: 5;
+  grid-column-end: 10;
```
Comment (kisp) [inline css-grid_place-elements/css/styles.css:54]: Try to use `-1` here to specify the end (regardless of how many columns there are). :slightly_smiling_face:

**Review by kisp — CHANGES_REQUESTED** [review]
Hi Markus, great work! :slightly_smiling_face: Mainly to try out the workflow, I am asking for a one line change. :slightly_smiling_face: 

We can do a ping / pong - after you have committed the change, I can review again and then approve the changes.

File: css-grid_place-elements/css/styles.css (hunk 1 above)
Comment (WinterIsCoding23) [reply css-grid_place-elements/css/styles.css]: Thank you very much for pointing this out!! I corrected it now.

File: css-grid_place-elements/css/styles.css [hunk 2]
```diff
@@ -51,7 +51,7 @@ body {
   background-color: #3a67a4;
   /* Add styles here */
   grid-column-start: 5;
-  grid-column-end: 10;
+  grid-column-end: -1;
```
Comment (kisp) [inline css-grid_place-elements/css/styles.css:54]: Glad, that you like it! :slightly_smiling_face:

**Review by kisp — APPROVED** [review]
Looks great! :+1:

**Pull request merged**